doc/source/train/key-concepts.rst

```diff
@@ -35,6 +35,8 @@ There are three categories of built-in Trainers:
     - :class:`TorchTrainer <ray.train.torch.TorchTrainer>`
     - :class:`TensorflowTrainer <ray.train.tensorflow.TensorflowTrainer>`
     - :class:`HorovodTrainer <ray.train.horovod.HorovodTrainer>`
+    - :class:`LightningTrainer <ray.train.lightning.LightningTrainer>`
+    - :class:`HuggingFaceTrainer <ray.train.huggingface.HuggingFaceTrainer>`
```

Comment:
nit: `HuggingFaceTrainer` is already included in "Other Trainers" - this is because it doesn't quite follow the user defined training loop pattern. We can duplicate it here for now, and revisit this in 2.5.

Reply:
Ok sure!